clients/intellij/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.1.1
```

Comment:
should be 1.1.2？

Reply:
I changed only vscode extension to 1.1.2.   
IntelliJ and vim plugin does not have a 1.1.1 version yet.